pilot/pkg/xds/discovery.go

```diff
@@ -162,8 +163,13 @@ type EndpointShards struct {
 	ServiceAccounts sets.Set
 }
 
+// GeneratorParams: Parameters for the various discover server generators
+type GeneratorParams struct {
```

Comment:
Why do we need a struct to hold the trustBundle ? It seems there is a single param, and we have already a place to hold  various discovery server parameters ( env )